model/src/commonMain/kotlin/io/github/droidkaigi/feeder/TimetableItem.kt

```diff
@@ -11,7 +11,7 @@ sealed class TimetableItem(
     open val endsAt: Instant,
     open val category: TimetableCategory,
     open val targetAudience: String,
-    open val language: String,
+    open val language: TimetableLanguage,
```

Comment:
I don't think it can be safely restricted by typealias. 👀 
![image](https://user-images.githubusercontent.com/1386930/134831753-bd5e419e-d7ca-4191-b86d-151818280e07.png)

Reply:
Sorry, this is my mistake. 😭